tests/Feature/Users/UserAdminTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Events\UserUpdated;
+use App\Role;
+use App\User;
+use Carbon\Carbon;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Tests\TestCase;
+use Illuminate\Support\Facades\Event;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+
+class UserAdminTest extends TestCase
+{
+    public function provider()
+    {
+        return [
+            [
+                'Administrator', TRUE
+            ],
+            [
+                'NetworkCoordinator', FALSE
+            ],
+            [
+                'Host', FALSE
+            ],
+            [
+                'Restarter', FALSE
+            ],
+        ];
+    }
+
+    /**
+     *@dataProvider provider
+     */
+    public function testUsersPage($cansee) {
+        // Fetch the list of all users and check that we're in it.
+        $admin = factory(User::class)->states('Administrator')->create();
```

Comment:
Seems like this test is expecting to check this for the various users, based on the data provider, but only seems to be testing it for an admin?

Reply:
You're right.  Fixed.